seguridad.md

```
@@ -0,0 +1,643 @@
+# Security by User Story
+
+Security requirements and acceptance criteria specific to the LMS-CMS, derived from the user stories defined in [readme.md](./readme.md#5-user-stories).
+
+## Table of Contents
+
+1. [Scope and exclusions](#scope-and-exclusions)
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)